osidb/cc.py

```diff
@@ -430,30 +377,11 @@ def generate(self) -> Tuple[List[str], List[str]]:
         generate content
         """
         all_cc = self.affect_list2cc(self.flaw.affects.all())
-
-        if not self.old_flaw:
-            # on create we add all
-            # and remove nothing
-            add_cc = all_cc
-            remove_cc = []
-
-        elif self.old_flaw.embargoed and not self.flaw.is_embargoed:
-            # on unembargo we add and remove all but do not add
-            # already existing CCs because they are already there
-            add_cc = [cc for cc in all_cc if cc not in self.old_cc]
-            remove_cc = self.affect_list2cc(self.removed_affects)
-
-        else:
-            # on update we only add the CCs from new affects
-            # so we do not re-add people who removed themselves
-            # and remove whatever is not needed any more
-            add_cc = self.affect_list2cc(self.added_affects)
-            remove_cc = self.affect_list2cc(self.removed_affects)
-
-        # do not remove CCs which are being added from another source
-        remove_cc = [cc for cc in remove_cc if cc not in add_cc]
-
-        return add_cc, remove_cc
+        add_cc = [cc for cc in all_cc if cc not in self.old_cc]
+        # simplify the functionality by only adding people
+        # who where never added before (accumulating)
+        # and never removing anyone - they can do it
+        return add_cc, []
```

Comment:
If we aren't using the `remove_cc` array anymore, we can just return the `add_cc` instead of also returning an empty list that will be ignored on every call anyway

Reply:
You are completely right. I wanted to save a bit of work on adjusting tests etc. as I am in a bit of hurry. This functionality should hopefully go completely away soon enough so if you do not insist on cleaning it I would just leave it as it is for the rest of its days